### PR TITLE
KIM Integration - AWS Freemium and Azure Lite

### PR DESCRIPTION
**Pull request by jaroslaw-pieszka**
**Description**

Changes proposed in this pull request:

- Shoot data for Azure Lite
- Shoot data for AWS Freemium 

**Related issue(s)**

### CODE REVIEW

File: internal/process/provisioning/create_runtime_resource_step.go [hunk 1]
```diff
@@ -228,6 +228,24 @@ func (s *CreateRuntimeResourceStep) providerValues(operation *internal.Operation
 			MultiZone:              s.config.MultiZoneCluster,
 			ProvisioningParameters: operation.ProvisioningParameters,
 		}
+	case broker.FreemiumPlanID:
+		var freemiumProvider internal.CloudProvider
+		if operation.ProvisioningParameters.Parameters.Provider == nil {
+			//TODO - default provider?
+			return provider.Values{}, fmt.Errorf("provider not definded for freemium plan")
+		} else {
```
Comment (ralikio) [inline internal/process/provisioning/create_runtime_resource_step.go:236]: Can we move it outside of else? Because of return it is not necessary.

Comment (jaroslaw-pieszka) [reply internal/process/provisioning/create_runtime_resource_step.go]: `else` removed in the next push.

File: internal/provider/aws.go [hunk 2]
```diff
@@ -87,3 +90,29 @@ func (p *AWSTrialInputProvider) region() string {
...
 }
+
+func (p *AWSFreemiumInputProvider) Provide() Values {
+	machineType := DefaultOldAWSTrialMachineType
+	if p.UseSmallerMachineTypes {
+		machineType = DefaultAWSMachineType
+	}
+	region := p.region()
+
+	return Values{
```
Comment (ralikio) [inline internal/provider/aws.go:101]: There is a lot of inputs set here: https://github.com/kyma-project/kyma-environment-broker/blob/b4a26925fa1ae24f1f063e6d12e92146b6aed064/internal/provider/azure_provider.go#L197-L198. Is that the correct place that should server as a template of what to sent to RunimeResource?

File: internal/provider/azure.go [hunk 3]
```diff
@@ -93,3 +96,36 @@ func (p *AzureTrialInputProvider) region() string {
 	}
 	return DefaultAzureRegion
 }
+
+func (p *AzureLiteInputProvider) Provide() Values {
```
Comment (ralikio) [inline internal/provider/azure.go:100]: The same question as defined here: https://github.com/kyma-project/kyma-environment-broker/pull/976/files#r1700033908

**Review by ralikio — APPROVED** [review]

**Pull request merged**